planner/core/memtable_predicate_extractor.go

```diff
@@ -640,3 +640,58 @@ func (e *InspectionSummaryTableExtractor) Extract(
 	e.MetricNames = metricNames
 	return remained
 }
+
+// SlowQueryExtractor is used to extract some predicates of `slow_query`
+type SlowQueryExtractor struct {
+	extractHelper
+
+	SkipRequest bool
+	StartTime   time.Time
+	EndTime     time.Time
+	Enable      bool
```

Comment:
It's better to leave some comments on the `Enable` field.

Reply:
great, done.